guardian/workflow-frontend
# Prepare for rollout of "Key Takeaways" and "Q and A Explainer" articles

## What does this change?

In preparation for roll out...

- removes feature switches
- updates icons
- renames "Q and A" to "Q and A Explainer"

Similar to this composer PR: https://github.com/guardian/flexible-content/pull/4744

## How to test

Run locally or deploy to CODE. Hit `shift-f12`. Ensure the switches for "Key Takeaways" and "Q and A Explainer" have gone. Look in the "Create new" menu. Can you create "Key Takeaways" and "Q and A Explainer" articles?

## Images

![Screenshot 2024-04-30 at 16 52 12](https://github.com/guardian/workflow-frontend/assets/7423751/f7094bb7-6c38-4b64-a485-aa37cb3be82c)

![Screenshot 2024-04-30 at 16 58 22](https://github.com/guardian/workflow-frontend/assets/7423751/bb03a766-c83a-4c0a-a025-c0b904db82a5)
